source/gui/_addonStoreGui/controls/details.py

```diff
@@ -222,6 +223,22 @@ def _refresh(self):
 					pgettext("addonStore", "Publisher:"),
 					details.publisher
 				)
+				if isinstance(details, InstalledAddonStoreModel):
+					# The author and publisher fields are both available in this case.
+					# Publisher comes from add-on store data, author comes from the manifest.
+					# For externally installed add-ons, the publisher field uses the author manifest field.

```

Comment:
If I understand correctly (not tested), in case of external add-ons, only the publisher field is displayed. Wouldn't it be clearer to use only author field instead? Things would be simpler:
* author = the author as advertised in the manifest file of the add-on for all channels, as before in NVDA 2023.1
* Publisher = the name indicated when submitting the add-on

Reply:
Unfortunately this is not simpler - as we expect a publisher field in these scenarios:
- as a column in the list of add-ons
- searching add-ons

Reply:
I understand your point but I find it misleading to have a different "Author" information wether the add-on is installed from the store or externally. This may lead to incomprehension when discussing installed add-ons.

May we use an "Author" column instead of "Publisher" in installed add-on tab and incompatible installed add-on tab? For updatable add-ons the question is open but I would slightly prefer "Publisher".

Regarding add-on search, both author and publisher fields should be considered. Add-on id should also be considered when searching; I have not checked it but I hope it is the case in this PR.

Reply:
Thanks for the suggestions - these have been implemented now